.gitignore

```diff
@@ -78,7 +78,7 @@ demo
 config/deploy/local/**/*.kubeconfig
 config/deploy/local/**/*.env
 config/deploy/**/_*
-
+config/istio/
```

Comment:
what's the reason for this?
The contents of that dir are checked in and used for local-setup

Reply:
a bunch of files I didn't create myself just showed up, so I didn't want to commit them. I can remove it, if it shouldn't be here.

Reply:
Yeah, let's remove.
I suspect it's from a change that removed some istio subdirs that landed on main recently, and your local git didn't remove them when switching branches.
They are save to manually `rm` if they still show up locally

Reply:
ok, I've removed that commit now. Thanks I'll do that.